### DISCUSSION TITLE
Caddy not working, ports open but cannot access trough my URL

### DISCUSSION THREAD

**Discussion by gatodelcaribe** [plausible/analytics#4549] [Self-Hosted Support]
Hi, I have 443 and 80 ports with access, caddy doesn't work making the reverse proxy I followed the guide on https://github.com/plausible/community-edition?tab=readme-ov-file creating it on Docker. 

I can access as local to http://127.0.0.1:8000 but the plausible.mydomain.com doesn't work.

**Comment by ruslandoga** [comment]
👋 @gatodelcaribe 

Would you be able to provide the real URL? Also what do `docker ps` and `docker logs <caddy container id>` print out?

**Reply by gatodelcaribe** [reply]
I think it's fixed, the port was opened only as TCP not UDP and TCP.
